app/src/page/file/view.js

```diff
@@ -177,7 +181,7 @@ class FilePage extends React.PureComponent {
       [
         { text: 'No' },
         { text: 'Yes', onPress: () => {
-          stopDownload(navigation.state.params.uri, fileInfo);
+          stopDownloadz4(navigation.state.params.uri, fileInfo);
```

Comment:
Where is this method coming from?
I do not see it added in this PR and a quick search did not yeild results

Reply:
Looks like a typo. Will fix.